conceptpower/classes.py

```diff
@@ -157,4 +158,60 @@ def get_type(self, uri):
                 data['supertype_id'] = snode.get('supertype_id')
                 data['supertype_uri'] = snode.get('supertype_uri')
         return data
-        
\ No newline at end of file
+
+    def create(self,user,password,required_concept_data,**kwargs):
```

Comment:
Single space after each comma.

Also, each required element should be a separate argument here, e.g.

```
def create(self, user, password, word, pos, description, type .... 
```